task7/es6.js

```diff
@@ -0,0 +1,320 @@
+(function () {
+        class PostCollection {
+
+            constructor(_posts) {
+                this._posts = _posts;
+            }
+
+            clear() {
+                this._posts.splice(0);
+            }
+
+            getInd(id) {
+                for (var i = 0; i < this._posts.length; i++) {
```

Comment:
Pls, use built-in array methods and avoid manual iteration over it (findIndex can be used here)